packages/experimental/src/components/EmptyState/EmptyState.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * Copyright IBM Corp. 2020, 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button } from 'carbon-components-react';
+
+import { expPrefix } from '../../global/js/settings';
+
+export const EmptyState = ({
+  actionText,
+  actionType,
+  actionIcon,
+  heading,
+  subtext,
+  illustration,
+  illustrationSize,
+  illustrationTheme,
+  onActionEvent,
+}) => {
+  return (
+    <div className={`${expPrefix}-empty-state`}>
+      {illustration && typeof illustration === 'string' && (
+        <img
+          src={require(`./EmptyStateIllustrations/${illustrationTheme}/${illustration}.svg`)}
```

Comment:
Please remove use of require.

Reply:
Hey Lee, so I did it this way to prevent having to import all 12 svgs in the component and only ever using 1 of them. If I remove this line, we will have to add 12 import statements for each svg version to the top of this component. And we'll also end up with a bunch of eslint errors such as...
```
'noDataSvg' is declared but its value is never read.ts(6133)
'noDataSvg' is defined but never used. Allowed unused vars must match /^_/u.eslintno-unused-vars
```
Is it ok to keep this or are you thinking about another way?

Reply:
I hadn't realised dynamic imports were not a thing yet https://github.com/tc39/proposal-dynamic-import

While we could put a placeholder image in a folder and strip off its name and construct the path that way it would seem to be a bit excessive.

Leave as is and if there is a better pattern found it can be added later.